app/controllers/documents_controller.rb

```diff
@@ -23,6 +23,11 @@ def show
 
   def update
     document = Document.find_by_param(params[:id])
+
+    if document.publication_state == "sent_to_live"
+      document.current_edition_number = document.current_edition_number + 1
```

Comment:
Can we do `document.current_edition_number += 1` here?

Reply:
👍 done

Reply:
Turns out `increment` exists as well! https://apidock.com/rails/ActiveRecord/Persistence/increment